### PR TITLE
feat(crons): Update SDK docs for empty check-ins

### PR DESCRIPTION
**Pull request by evanpurkhiser**
See https://github.com/getsentry/sentry/issues/49664

### CODE REVIEW

File: src/docs/sdk/check-ins.mdx [hunk 1]
```diff
@@ -24,6 +24,11 @@ The following fields exist:
 
 : **String, required**. Check-In ID (unique and client generated).
 
+  This may be provided as a empty UUID (128 bit zero value) to indicate to
```
Comment (cleptric) [inline src/docs/sdk/check-ins.mdx:27]: Do you expect the SDKs to implement this feature?

Comment (evanpurkhiser) [reply src/docs/sdk/check-ins.mdx:27]: At the moment no I would expect almost all SDKs to maintain their checkin IDs for each run. This is to support the simplified CURL style API as outlined in https://github.com/getsentry/sentry/issues/49664

Comment (cleptric) [reply src/docs/sdk/check-ins.mdx:27]: The linked PR mentions `null`, do you suggest `00000...`?

File: src/docs/sdk/check-ins.mdx [hunk 2]
```diff
@@ -24,6 +24,11 @@ The following fields exist:
...
+  Sentry that the checkin should update the previous check-in that is not in
+  the terminal state. If the most recent check-in is already in a terminal
+  state a new one will be created instead.
```
Comment (cleptric) [inline src/docs/sdk/check-ins.mdx:30]: It might be worth clarifying what terminal state means.

File: src/docs/sdk/check-ins.mdx (hunk 1 above)
Comment (evanpurkhiser) [reply src/docs/sdk/check-ins.mdx:27]: I think @mitsuhiko's description is slightly misleading, it is not nullable, but it passes a empty value instead. You can see it in the tests here https://github.com/getsentry/relay/pull/2153/files#diff-2b7b3259fa82d6066fa1268aed58cf7a8bac4e9317e275b7601cc916b1201e4bR56

File: src/docs/sdk/check-ins.mdx (hunk 2 above)
Comment (rjo100) [reply src/docs/sdk/check-ins.mdx:30]: yeah reverse the logic so it should "update the most recent check-in in an `in progress` state. If the most recent check-in is not `in progress`..."

**Pull request merged**